core/src/test/scala/com/salesforce/op/stages/impl/feature/OpScalarStandardScalerTest.scala

```diff
@@ -148,6 +150,35 @@ class OpScalarStandardScalerTest extends OpEstimatorSpec[RealNN, UnaryModel[Real
     assert(sumSqDist <= 0.000001, "===> the sum of squared distances between actual and expected should be zero.")
   }
 
+  it should "descale and work in standardized workflow" in {
+    val featureNormalizer = new OpScalarStandardScaler().setInput(testF)
+    val normedOutput = featureNormalizer.getOutput()
+    val metadata = featureNormalizer.fit(inputData).getMetadata()
+    val expectedStd = 90.0 * math.sqrt(37.0)
+    val expectedMean = 370.0
+    val expectedSlope = 1 / expectedStd
+    val expectedIntercept = expectedMean / expectedStd
+    ScalerMetadata(metadata) match {
+      case Failure(err) => fail(err)
+      case Success(meta) =>
+        meta shouldBe a[ScalerMetadata]
+        meta.scalingType shouldBe ScalingType.Linear
+        meta.scalingArgs shouldBe a[LinearScalerArgs]
+        meta.scalingArgs.asInstanceOf[LinearScalerArgs].slope - expectedSlope should be < 0.001
+        meta.scalingArgs.asInstanceOf[LinearScalerArgs].intercept - expectedIntercept should be < 0.001
+    }
+
+    val descaledResponse = new DescalerTransformer[RealNN, RealNN, RealNN]()
+      .setInput(normedOutput, normedOutput).getOutput()
+    val workflow = new OpWorkflow().setResultFeatures(descaledResponse)
+    val wfModel = workflow.setInputDataset(inputData).train()
+    val transformed = wfModel.score()
+
+    val actual = transformed.collect().map(_.getAs[Double](1))
+    val expected = Array(-730.0, -640.0, 260.0)
```

Comment:
Same here, could we calculate the expected from the raw data so we'd only have to change one thing in the future?